client/my-sites/importer/error-pane.jsx

```diff
@@ -97,6 +97,17 @@ class ImporterError extends PureComponent {
 			);
 		}
 
+		if ( this.props.code === FileTooLarge.FILE_TOO_LARGE ) {
+			return this.props.translate(
+				'The file you are importing is too large. {{cs}}Please contact support to continue{{/cs}}.',
```

Comment:
Copy could be improved.

Reply:
We could also put a little text under the drop zone that tells the user the maximum size of the file. Otherwise i agree that it would be nice to let the user know the max size via the error message.

Reply:
Feel free to add! Adding the max file size is simple, carrying over the file size they tried to upload might be little more work. A helpful utility function:

https://github.com/Automattic/wp-calypso/blob/854eb8f01b04317d7d204f9f7352a8926bc06d50/client/my-sites/backup/backup-contents-page/file-browser/util.ts#L177-L187